### DISCUSSION TITLE
Cypress 10 component testing and Tailwindcss

### DISCUSSION THREAD

**Discussion by hsheikhali1P** [cypress-io/cypress#22095] [Component Testing]
Is there a way to have Cypress 10 component testing work with Tailwindcss ? I haven't been able to find a reliable guide.. As of right now my components have 0 styles and I think that's because cypress isn't running anything to let the project know that there are tailwind classes being used?

**Comment by marktnoonan** [comment]
Hi, there is some reference to this in the [docs about styling components](https://docs.cypress.io/guides/component-testing/styling-components). In the simplest case you can add a global stylesheet like tailwind to `cypress/support/component-index.html`, which provides the mounting context for your components. 

Another option is to import CSS in your component testing support file, located at `cypress/support/component.ts`. Styles imported there will get applied also, so it depends a bit on your setup.

**Comment by mikespinks0401** [comment]
I followed the tailwind instructions [here ](https://tailwindcss.com/docs/installation/using-postcss) to get Vite to pick up the styles
